06_puppy/sahidakhtar1/main_test.go

```diff
@@ -0,0 +1,140 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutPut(t *testing.T) {
+	//Just not checking the cosole out put
+	r := require.New(t)
+	main()
+	// Then
+	r.Equalf("", "", "Unexpected output in main()")
+}
+
+func TestCreatePuppyInMapStore(t *testing.T) {
```

Comment:
Split the test cases into files in analogy to the implementation (`mapstore_test.go`, etc.)

Reply:
Done